pallas-primitives/src/alonzo/json.rs

```diff
@@ -62,8 +62,8 @@ impl ToCanonicalJson for super::NativeScript {
                 let scripts: Vec<_> = k.iter().map(|i| i.to_json()).collect();
                 json!({ "type": "atLeast", "required": n, "scripts" : scripts })
             }
-            super::NativeScript::InvalidBefore(slot) => json!({ "type": "before", "slot": slot }),
-            super::NativeScript::InvalidHereafter(slot) => json!({"type": "after", "slot": slot }),
+            super::NativeScript::InvalidBefore(slot) => json!({ "type": "after", "slot": slot }),
+            super::NativeScript::InvalidHereafter(slot) => json!({"type": "before", "slot": slot }),
```

Comment:
wait it seemed correct before? is this one of those situations where it's unintuitive, so like on `InvalidBefore` we want to say `after this slot`?

Reply:
@SmaugPool nice I see. Thank you